onnx/reference/ops/op_reduce_l1.py

```diff
@@ -3,11 +3,46 @@
 
 import numpy as np
 
+from onnx.defs import onnx_opset_version
+
 from ._op import OpRunReduceNumpy
 
 
-class ReduceL1(OpRunReduceNumpy):
+class ReduceL1_1(OpRunReduceNumpy):
     def _run(self, data, axes=None, keepdims=None):  # type: ignore
         return (
             np.sum(np.abs(data), axis=axes, keepdims=keepdims).astype(dtype=data.dtype),
         )
+
+
+class ReduceL1_11(ReduceL1_1):
+    pass
+
+
+class ReduceL1_13(ReduceL1_1):
+    pass
+
+
+class ReduceL1_18(OpRunReduceNumpy):
+    def run(self, data, axes=None):  # type: ignore
```

Comment:
I suggest adding `noop_with_empty_axes=False` as a parameter, and passing it to `_run`.